packer/scripts/macos/macos-agentsetup.sh

```diff
@@ -22,7 +22,14 @@ cd MacPorts-2.7.2
 cd .. && rm -rf MacPorts-2.7.2.tar.gz
 export PATH=/opt/local/bin:$PATH
 sudo port -v selfupdate
+yes | sudo port install openjdk8-temurin
 yes | sudo port install openjdk11-temurin
+yes | sudo port install openjdk14-temurin
+yes | sudo port install openjdk17-temurin
+yes | sudo port install openjdk19
+yes | sudo port install openjdk20
+yes | sudo port install openjdk21-temurin
+export JAVA_HOME=/Library/Java/JavaVirtualMachines/openjdk11-temurin/Contents/Home
```

Comment:
This does not preserve over time at all.
Only exist in the life cycle of the script.

Reply:
thanks, Peter .. I am changing to use jEnv to set the Java version to jdk11 .. looks like jEnv is widely used to switch between Java versions https://github.com/jenv/jenv